### DISCUSSION TITLE
ETC_RGB4 and ASTC_RGB_6x6 texture encoding

### DISCUSSION THREAD

**Discussion by FrothierNine346** [nesrak1/AssetsTools.NET#139] [Q&A] [answered]
Hi,

I’ve noticed that the main branch of the project currently doesn’t support ETC_RGB4 and ASTC_RGB_6x6 texture encoding. However, it seems the dev branch already includes support for these formats. Should I use the dev branch for now if I need these features? Also, do you have an estimated timeline for the next update to the main branch?

Thank you!

**Comment by nesrak1** [comment, marked as answer]
Yes, you should use the dev branch if you want encoding for anything more than simple RGBA formats. Originally, the texture encoding work was going on in the UABEA project, but it's since moved here so you don't have to worry about importing UABEA code just for some basic texture encoding.
